### ISSUE TITLE
Text the app is broken

### ISSUE THREAD

**Issue by zanemcca** [BranchMetrics/smart-app-banner-branch-deep-linking-web-library#330]
The link is not being parsed correctly. A fix is already in the code it just needs to be put into the release.
The fix is the addition of `+ "/"` [here](https://github.com/BranchMetrics/smart-app-banner-branch-deep-linking-web-library/blob/master/dist/build.js#L1711)

**Comment by aaustin** [comment]
Thanks for this @zanemcca - We'll get this patched first thing tomorrow.

**Comment by aaustin** [comment]
Just tested on https://cdn.branch.io/example.html and the link came through fine - can you share a page you're working on or the link you're testing with so that we can try to reproduce? Thanks!

**Comment by zanemcca** [comment]
Sure @aaustin https://www.instanews.com/news/feed?loc=45.5176,-73.5804,6z should have the problem but unfortunately you do not have access to the `text me the app` feature unless you are in Canada. I think the key difference is that I have a custom domain name. So on this [line of code](https://github.com/BranchMetrics/smart-app-banner-branch-deep-linking-web-library/blob/master/dist/build.js#L1711) it ends up with `https://bnc.ltrucn/eSPKd4UkOt` instead of `https://bnc.lt/rucn/eSPKd4UkOt`

**Comment by jsaleigh** [comment]
Hi @zanemcca !  Thanks for the report on this.  There is definitely something odd in here, so let me walk through my findings, since I may need your help:

- The source you pointed out with the fix is for v2.2.0 which was released on May 5th (about 3 weeks ago)
- Your code appears to have `branch-sdk` from npm bundled in at this endpoint: https://www.instanews.com/scripts/vendor.js
- The `branch-sdk` version in your code is v2.2.0, so that looks great!
- Searching for `/(bnc.lt\/|app\.link\/)/`in the current dist/build.min.js, and at the CDN version https://cdn.branch.io/branch-latest.min.js, both result in the correct code `(d="https://bnc.lt/"+ia(d))`
- However, searching for the same regex in your bundle at https://www.instanews.com/scripts/vendor.js shows incorrect code: `(d="https://bnc.lt"+ia(d))`.  Notice that the trailing slash is missing in there, for some reason.

It seems like either something in your build process is stripping the trailing slash, or the code from your version of npm is corrupted.

In either case, I would like to be able to handle this from within the sdk itself.  So any information you can give us would be wonderful.

Would you be willing to check your `node_modules\branch-sdk\dist\build.min.js` file, or whichever source file you are using, to see if the trailing slash exists there?  This is just to narrow down the cause.

**Comment by zanemcca** [comment]
@jsaleigh Thanks for having a look.

I performed the same search before reporting and discovered the discrepancy. I think what might be the difference between our setups is that I am using bower for the install not npm. 
When I check `bower_components/branch-sdk/dist/build.min.js` it does not have the change in it even though `bower_components/branch-sdk/bower.json` states that the version is 2.2.0. 

I noticed that the fix was done in this commit c8b3f9fbbf9e4f9f27d97c1f22a6457b5b24513a 2 days after this commit 9888b24626e281d5ab7c0ff6baca4a36e306d7db which updated the bower version. Also the 2.2.0 tag was created 2 days before the update. So assuming you guys are using the automated bower updates it would have missed the latest changes.
